src/main/java/com/example/demo/controllers/UsersController.java

```diff
@@ -23,52 +24,88 @@
 public class UsersController {
 
     @Autowired
-    private StudentRepository userRepo;
+    private UserRepository userRepo;
 
-    @GetMapping("/users/view")
-    public String getAllUsers(Model model)
+    //simple get request to login page
+    @GetMapping("/login")
+    public String loginPage() {
+        return "users/loginPage";
+    }
+    @GetMapping("/register")
+    public String registerPage() {
+        return "users/registerPage";
+    }
+
+    @PostMapping("/users/login")
+    public String login(@RequestParam Map<String, String> newuser, HttpServletResponse response, Model model)
```

Comment:
Just a small detail I noticed, do we have a specific reason to not use camel case on newuser? (so newUser instead of newuser). Would it be okay to refactor all newuser occurences to newUser?